.github/workflows/insiders.yml

```diff
@@ -584,7 +584,7 @@ jobs:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
```

Comment:
YOu have to remove the azure credential login too. Otherwise PRs from other forks fail.